addon/ng2/tasks/serve-webpack.ts

```diff
@@ -15,7 +15,8 @@ module.exports = Task.extend({
     let lastHash = null;
     let webpackCompiler: any;
 
-    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment).config;
+    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment, commandOptions.outputPath).config;
```

Comment:
Content base property for webpack dev server needs to be set also. This I believe is pulled in from webpack-build-utils.